docs/docsource/operation/Zenko_CLI/managing_bucket_policies.rst

```diff
@@ -0,0 +1,18 @@
+.. _managing bucket policies:
+
+Managing Bucket Policies
+========================
+
+In addition to controlling access using access control list (ACL) policies
```

Comment:
The proper term here would be access control lists (they're technically not policies) and they're not user-based :)

Reply:
Thanks! Will fix.

Reply:
I'd also group CORS in with ACLs at the beginning of the sentence, sth like "In addition to controlling access using access control lists (ACLs) and cross-origin resource sharing (CORS), bucket owners can control access at the resource level using bucket policies."

Reply:
done.